app/views/new_threads/list.html.erb

```diff
@@ -1,87 +1,61 @@
-<b><%= link_to("New Thread", {:action => 'new'}, :class => 'navbar-link new') %></b><br><br>
-
-<% if user_signed_in? %>
-  <span class="user other-user"> <%= current_user.username %></span>
-  <%= image_tag current_user.image.url(:thumb), size: "50x50", :class => 'user-img other-img'%>
-<% end %>
-
-<% if !user_signed_in? %>
-  <p class=" welcome thread-welcome">
-    Welcome to our CAT Forum!
-    Not a member yet? Why don't you click on sign up, it will only take a few minutes. You can log in with Facebook too. 
-    Browse through the our list of recent threads, We are sure you will find something interesting.
-  </p>
+<div id="new-thread-button" class="btn btn-primary">
+<%= link_to({:action => 'new'}, :class => 'navbar-link new') do %>
+<span class="glyphicon glyphicon-pencil" aria-hidden="true"></span>
+<span>New Thread</span>
 <% end %>
+</div>
 
-<p class="recent">
-  Recent Threads:
-</p>
-
-<div class="thread-whole">
-  <% @new_threads.each do |new_thread| %>
-    <div itemscope itemtype="http://schema.org/Question">
-      <%= image_tag new_thread.user.image_url(:thumb).to_s, :class => "user-image" %>
-      <%= image_tag("arrow.png", class: "triangle fade_speech") %>
-      <div class="speech-bubble fade_speech">
-        <h1><%= link_to new_thread.title, new_thread, :class => "show-thread", :itemprop=>"url"%></h1><br>
-        <span class="time">asked <time itemprop="dateCreated" datetime=<%= new_thread.created_at %>><%= time_ago_in_words(new_thread.created_at) %></time> ago</span><br><br><br>
-        <h2><span class="description pt-page-moveToLeft" itemprop="text"><%= new_thread.description %></span></h2><br><br><br><br>
-        <div class="info">
-          <table>
-            <tr>
-              <td>
-                <% @new_thread = new_thread %>
-                <%= render :partial => "like_counts/new_thread", :locals => {:new_thread_id => @new_thread.id} %>
-              </td>
-              <td>
-                <div class="like">
-                  <%= button_to 'Like', like_counts_path(:new_thread_id => new_thread), remote: true, :class => "like-bu"%>
-                </div>
-              </td>
-              <td>
+<div class="thread-list-wrapper">
+  <div class="container thread-list">
+    <div class="row thread-row">
+      <% @new_threads.each do |new_thread| %>
+        <% @new_thread = new_thread %>
+        <div class="col-xs-12 col-md-6 col-lg-4 thread-col">
```

Comment:
@ghoshbishakh i think some inline css are auto generated.i can see some inline css in this line in the source code.